docs/GETTING_STARTED_GUIDE.md

```diff
@@ -22,6 +22,23 @@ First and foremost, the metrics service requires a Kubernetes cluster that align
 | --------- |
 | 1.17-1.19 |
 
+## Supported Dell EMC Products
+
+This project currently supports the following Dell EMC storage systems.
+
+| Dell EMC Storage Product |
+| ----------------------- |
+| PowerFlex v3.0/3.5 |
+
+
+## CSI Driver for Dell EMC PowerFlex
+
+This project captures telemetry data about Kubernetes storage usage and performance which are provisioned through the CSI (Container Storage Interface) Driver for Dell EMC  PowerFlex. The metrics service requires that the CSI Driver for Dell EMC PowerFlex is deployed in the Kubernetes cluster.
```

Comment:
I think that there is an extra space between Dell EMC an PowerFlex. "Dell EMC  PowerFlex."